src/pivot-table/hooks/__tests__/use-selections-model.test.ts

```diff
@@ -44,39 +52,66 @@ describe("useSelectionsModel", () => {
   });
 
   test("should select cell and call begin selection", async () => {
-    const { result } = renderHook(() => useSelectionsModel(selections));
+    const { result } = renderHook(() => useSelectionsModel(selections, updatePageInfo));
 
     await act(async () => {
+      Object.entries(callbacks).forEach(([, cb]) => cb());
```

Comment:
I still don't think this fits in this test. It is only testing what happens when you select a cell and that begin should be called. It does not test the listeners that you attach with `on`. It makes more sense to me if we have a seperate test, looping over all callbacks.